### PR TITLE
Add Eppo integration tile

### PR DESCRIPTION
**Pull request by emetelka**
### What does this PR do?

Adds an integration tile for [Eppo](https://www.geteppo.com/)

### Motivation

What inspired you to submit this pull request?

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)
- [ ] If this PR includes a log pipeline, please add a description describing the remappers and processors. 

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

**Comment by drichards-87** [comment]
Created Jira card for Docs Team editorial review.

File: eppo/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Agent Check: Eppo / Datadog RUM
+
+## Overview
+
+[Eppo][1] is the experimentation and feature management platform that makes advanced A/B testing accessible to everyone in your organization.
+
+The Datadog Eppo RUM integration enriches your RUM data with your feature flags to provide visibility into performance monitoring and behavioral changes. Determine which users are shown a user experience and if it is negatively affecting the user's performance.
```
Comment (emarsha94) [inline eppo/README.md:7]: Will users see this in Datadog or Eppo? Can you clarify what specifically is sent to Datadog?

File: eppo/README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+await eppoInit({
+  apiKey: "<API_KEY>",
+  assignmentLogger,
+});
+```
+
+## Troubleshooting
+
+Need help? See the [Eppo documentation][3] or contact [Datadog Support][4].
```
Comment (emarsha94) [inline eppo/README.md:41]: This should be Eppo's support team

File: eppo/images/IMAGES_README.md
Comment (emarsha94) [inline eppo/images/IMAGES_README.md:1]: This can be deleted

File: eppo/assets/dashboards/eppo_overview.json
Comment (emarsha94) [inline eppo/assets/dashboards/eppo_overview.json:1]: Please include a dashboard

**Review by emarsha94 — CHANGES_REQUESTED** [review]

File: eppo/CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# CHANGELOG - Eppo
+
+## 1.0.0 / 2024-04-04
```
Comment (emarsha94) [inline eppo/CHANGELOG.md:3]:
```suggestion
## 1.0.0 / 2024-04-11
```

**Review by emarsha94 — APPROVED** [review]
Thank you for making the changes requested! Last thing is to update your changelog date then we are good to go!

**Review by cswatt — APPROVED** [review]
LGTM! I left one minor suggestion.

File: eppo/CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# CHANGELOG - Eppo
+
+## 1.0.0 / 2024-04-13
```
Comment (emarsha94) [inline eppo/CHANGELOG.md:3]:
```suggestion
## 1.0.0 / 2024-05-08
```

**Review by emarsha94 — APPROVED** [review]

**Review by JoshPatel13 — APPROVED** [review]

**Pull request merged**